src/components/feature-coverage/FeatureCoverage.tsx

```diff
@@ -55,10 +61,13 @@ const columns: ColumnDef<any>[] = [
   {
     id: 'k8s_support',
     accessorFn: (row) => row[Object.keys(row)[0]].k8s_test_suite,
-    header: () => 'Tested on Kubernetes',
+    header: () => 'Verified on Kubernetes',
```

Comment:
For other reviewers: this is a crucial part of the motivation for this change. We want to add clarity on what this column means, and the title should make it clear, and the tooltip provides extra context.